app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -115,8 +155,14 @@ ProjectBoard.propTypes = {
   fetchProjectBoard: PropTypes.func.isRequired,
   createStory: PropTypes.func.isRequired,
   closeHistory: PropTypes.func.isRequired,
-  notifications: PropTypes.array.isRequired
-}
+  notifications: PropTypes.array.isRequired,
+  reverseColumns: PropTypes.func,
```

Comment:
We should now have a 'defaultProps' object right?

Reply:
Actually all the props of this components should be required. I forgot that, thanks!